commit 81660b26bc8e2c03f55cf928955f7ed84e00785a
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Fri Oct 31 10:17:07 2008 +0200

    Adjust for file output change (rhbz#468129)
    - file util used reported file details along with mime type, newer ones don't

diff --git a/scripts/find-debuginfo.sh b/scripts/find-debuginfo.sh
index 287c512..21366bf 100644
--- a/scripts/find-debuginfo.sh
+++ b/scripts/find-debuginfo.sh
@@ -90,7 +90,7 @@ strip_to_debug()
 {
   local g=
   $strip_g && case "$(file -bi "$2")" in
-  application/x-sharedlib,*) g=-g ;;
+  application/x-sharedlib*) g=-g ;;
   esac
   eu-strip --remove-comment $g -f "$1" "$2" || exit
 }
